packages/design-system-docs/src/pages/components/Pagination/_Pagination.docs.scss

```diff
@@ -46,8 +69,15 @@ The following Sass variables can be overridden to theme alerts:
 - `$pagination-page-count` - Used to set the page count element text color.
 
 ### Accessibility
-#### Keyboard support
 
+- Use a wrapping `<nav>` element to identify Pagination as a navigation section.
+- Pagination **must** have an ARIA label attribute defined on the `<nav>` element that describes its purpose.
+- Use an unordered list for the navigation items as this allows screen readers to voice the number of elements in the Pagination component.
```

Comment:
[suggestion] this documentation says to use an `unordered` list, but the actual HTML is using an ordered list element. Consider updating either the documentation or the HTML to be consistent